src/main/java/application/rest/StackUtils.java

```diff
@@ -327,9 +354,24 @@ public static List streamLineMasterMap(List<Map> list) {
 		return aList;
 	}
 	
+	private static HashMap<String,String> getKabStackDigest(Stack s, String versionToFind) {
+		String digest=null,imageName=null;
+		HashMap<String,String> imageMetaData = new HashMap<String,String>();
+		
+		List<StackStatusVersions> versions=s.getStatus().getVersions();
+		for (StackStatusVersions version:versions) {
```

Comment:
add comment for when we have multiple images, potentially for multiple architectures

Reply:
comment added